js/everything.js

```diff
@@ -279,6 +281,17 @@ function getVideoDescription(videoID) {
     });
 }
 
+function highLightURL(text) {
+    /* RegEx to match http or https addresses
+    * This will currently only match TLD of two or three letters
+    * Ends capture when:
+    *    (1) it encounters a TLD
+    *    (2) it encounters a period (.) or whitespace, if the TLD was followed by a forwardslash (/) */
+    var re = /((?:http|https)\:\/\/[a-zA-Z0-9\-\.]+.[a-zA-Z]{2,3}(?:\/\S*[^\.\s])?)/g;
```

Comment:
This doesn't appear to work as is for our "demo" video description. If I change the last `?` to `+` it works fine - thoughts?

You can try it on the following text:
```
OUT NOW! Download the new EP "Piece Of The Indestructible" here: http://radi.al/Indestructible

TOUR DATES: http://theglitchmob.com/tour
9/18-20 - Sacramento, CA - TBD Festival
12/31 - Birmingham, Alabama - Bassnectar NYE 360

Vinyl. Merchandise and more here: http://smarturl.it/TGMstore

The Glitch Mob
http://fb.me/theglitchmobmusic
http://twitter.com/theglitchmob
http://instagram.com/theglitchmob
http://soundcloud.com/theglitchmob
http://youtube.com/glitchmob
SNAPCHAT: theglitchmob
```

Before:

![capturfiles_5](https://cloud.githubusercontent.com/assets/1480063/11555217/71bb0176-9953-11e5-9557-5335c9b021a4.png)

After:

![capturfiles_6](https://cloud.githubusercontent.com/assets/1480063/11555218/71be6e38-9953-11e5-9261-97d43ef73265.png)